pkg/writer/pointerdos.go

```
@@ -0,0 +1,155 @@
+		metadataKeysStandard,
+	)
+	return buf[:pos]
+}
+
+// WritePointerDecoderDoSTestDB writes the databases that exercise the
+// data-section pointer fan-out denial of service: a minimal single-node
+// database and a conventional IPv6 database that maps all of the address space
+// to the fan-out record. See buildPointerFanOutData.
+func (w *Writer) WritePointerDecoderDoSTestDB() error {
+	files := map[string][]byte{
+		"MaxMind-DB-test-pointer-decoder-dos.mmdb": buildPointerFanOutDB(pointerDoSDepth),
```

golangci-lint (2.12.2)

[error] string `MaxMind-DB-test-pointer-decoder-dos.mmdb` has 3 occurrences, make it a constant

(goconst)